requests/api.py

```diff
@@ -18,7 +18,9 @@ def request(method, url, **kwargs):
 
     :param method: method for the new :class:`Request` object.
     :param url: URL for the new :class:`Request` object.
-    :param params: (optional) Dictionary or bytes to be sent in the query string for the :class:`Request`.
+    :param params: (optional) Dictionary or list of tuples
```

Comment:
This can probably be two lines instead of three, following the line length of the other entries.

Reply:
To use 2 lines, one needs to cut the ` ``literal`` ` block, and I've seen some RST processor yield incorrect results in this case so I preferred avoiding this cut. I can change it if you prefer.

Reply:
I don’t think we need to cut the literal block. This line is almost identical to the one below it which only occupies a single line.

Reply:
Indeed: my text editor wraps at 72 characters so it was cutting in the middle of the literal block. I make lines a bit longer, but in most cases it doesn't change the number of lines. Other docstrings fit on one line because they're e.g. 171 characters long.

Reply:
@moy yes, so if we split it across two lines it will be < 100 characters per line which is consistent with this projects typical line length.

Reply:
I'm a new contributor so I can't talk about the established practices, but the doc says "Line-length can exceed 79 characters, to 100, when convenient.", so I understand that I can make an exception to the 79 characters when I have a good reason ("when convenient"). I don't see the benefit of making an exception here.